{"uid":"2ffdc1063d9ea66b","name":"test_scaling","fullName":"tests.integration.ha_tests.test_async_replication#test_scaling","historyId":"516219b7444bea9da0cd675ba612670b","time":{"start":1751766987307,"stop":1751766987307,"duration":0},"description":"Test that async replication works when scaling the clusters.","descriptionHtml":"<p>Test that async replication works when scaling the clusters.</p>\n","status":"skipped","statusMessage":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 719, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 548, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 639, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1127, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"ops_test","time":{"start":1751765452429,"stop":1751765452663,"duration":234},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1751765452429,"stop":1751765452429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1751765452429,"stop":1751765452429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"second_model","time":{"start":1751765452663,"stop":1751765453309,"duration":646},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"first_model","time":{"start":1751765452663,"stop":1751765452663,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1751766987308,"stop":1751766987308,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Test that async replication works when scaling the clusters.","status":"skipped","statusMessage":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 719, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 548, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 639, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1127, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"862faa4420000ed3","name":"log","source":"862faa4420000ed3.txt","type":"text/plain","size":2142}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"second_model::finalizer","time":{"start":1751766987313,"stop":1751766987332,"duration":19},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1751766987333,"stop":1751766988377,"duration":1044},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1751766988378,"stop":1751766988378,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1751766988378,"stop":1751766988378,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1751766988378,"stop":1751766988378,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_async_replication"},{"name":"host","value":"pkrvmbietmlfzoi"},{"name":"thread","value":"19668-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_async_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"48fea085269ba1f4","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":3,"broken":3,"skipped":46,"passed":323,"unknown":8,"total":383},"items":[{"uid":"3473ea3d18486c5f","reportUrl":"../4239//#testresult/3473ea3d18486c5f","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1751680688596,"stop":1751680688596,"duration":0}},{"uid":"5e906b392d1f1ea2","reportUrl":"../4238//#testresult/5e906b392d1f1ea2","status":"passed","time":{"start":1751594290797,"stop":1751594799511,"duration":508714}},{"uid":"e55c6f4a50ba9831","reportUrl":"../4233//#testresult/e55c6f4a50ba9831","status":"passed","time":{"start":1751507946040,"stop":1751508775278,"duration":829238}},{"uid":"2fdb7396c28b5d61","reportUrl":"../4229//#testresult/2fdb7396c28b5d61","status":"unknown","time":{}},{"uid":"8ce332d4c5d21897","reportUrl":"../4218//#testresult/8ce332d4c5d21897","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1751334916652,"stop":1751334916652,"duration":0}},{"uid":"6aecadb36132734","reportUrl":"../4214//#testresult/6aecadb36132734","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1751248618163,"stop":1751248618163,"duration":0}},{"uid":"d15b01f71524e231","reportUrl":"../4213//#testresult/d15b01f71524e231","status":"passed","time":{"start":1751162356828,"stop":1751163106166,"duration":749338}},{"uid":"90dfe9dd07f876f9","reportUrl":"../4212//#testresult/90dfe9dd07f876f9","status":"passed","time":{"start":1751075930850,"stop":1751076620107,"duration":689257}},{"uid":"9abebaa10a2b6031","reportUrl":"../4211//#testresult/9abebaa10a2b6031","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750989414784,"stop":1750989414784,"duration":0}},{"uid":"f381dd57a0d470bb","reportUrl":"../4205//#testresult/f381dd57a0d470bb","status":"passed","time":{"start":1750903071250,"stop":1750903632513,"duration":561263}},{"uid":"94f78bdc7e9646c7","reportUrl":"../4193//#testresult/94f78bdc7e9646c7","status":"passed","time":{"start":1750816600066,"stop":1750817286551,"duration":686485}},{"uid":"c6d0995e1273d46c","reportUrl":"../4186//#testresult/c6d0995e1273d46c","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750730165402,"stop":1750730165402,"duration":0}},{"uid":"41f46ffffbe49617","reportUrl":"../4185//#testresult/41f46ffffbe49617","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750643840523,"stop":1750643840523,"duration":0}},{"uid":"5d0362283a57289","reportUrl":"../4184//#testresult/5d0362283a57289","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750557402600,"stop":1750557402600,"duration":0}},{"uid":"94551858fb66e252","reportUrl":"../4183//#testresult/94551858fb66e252","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750471616643,"stop":1750471616643,"duration":0}},{"uid":"5d9c1d7e617da986","reportUrl":"../4177//#testresult/5d9c1d7e617da986","status":"passed","time":{"start":1750384701093,"stop":1750385457681,"duration":756588}},{"uid":"fe9113655418c54d","reportUrl":"../4167//#testresult/fe9113655418c54d","status":"passed","time":{"start":1750298358792,"stop":1750299054632,"duration":695840}},{"uid":"a396bc46c2097d84","reportUrl":"../4159//#testresult/a396bc46c2097d84","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750211780641,"stop":1750211780641,"duration":0}},{"uid":"70ecdeab54ce4fae","reportUrl":"../4148//#testresult/70ecdeab54ce4fae","status":"passed","time":{"start":1750125496628,"stop":1750126068297,"duration":571669}},{"uid":"1aff4c12cc549ace","reportUrl":"../4140//#testresult/1aff4c12cc549ace","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1750041069451,"stop":1750041069451,"duration":0}}]},"tags":["asyncio","abort_on_fail"]},"source":"2ffdc1063d9ea66b.json","parameterValues":[]}